src/Foreign/JavaScript/CallBuffer.hs

```diff
@@ -5,6 +5,8 @@ import Control.Concurrent
 import Control.Concurrent.STM as STM
 import Control.Monad
 
+import GHC.Conc
```

Comment:
Importing a GHC-specific module is not a good idea. I would prefer to use functions from standard modules like `Control.Concurrent` only.